pom.xml

```diff
@@ -1673,6 +1673,13 @@
                 <version>${dep.jna.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>net.minidev</groupId>
+                <artifactId>json-smart</artifactId>
+                <!-- not vulnerable to CVE-2023-1370 -->
```

Comment:
No need for this comment. The mention in the commit message is sufficient. If we added a comment like this to every dependency, we would have an unbounded growing list.